pkg/provision/buildroot.go

```diff
@@ -149,20 +151,31 @@ func (p *BuildrootProvisioner) Provision(swarmOptions swarm.Options, authOptions
 	log.Debugf("set auth options %+v", p.AuthOptions)
 
 	log.Debugf("setting up certificates")
-
 	configureAuth := func() error {
 		if err := configureAuth(p); err != nil {
 			return &util.RetriableError{Err: err}
 		}
 		return nil
 	}
-
 	err := util.RetryAfter(5, configureAuth, time.Second*10)
 	if err != nil {
 		log.Debugf("Error configuring auth during provisioning %v", err)
 		return err
 	}
 
+	log.Debugf("setting minikube options for container-runtime")
+	smo := func() error {
+		if err := setMinikubeOptions(p); err != nil {
+			return &util.RetriableError{Err: err}
+		}
+		return nil
+	}
+	err = util.RetryAfter(5, smo, time.Second*10)
```

Comment:
Not sure you need this many retries - configureAuth only has them because it's the first time we setup and use SSH AFAIK. After this you should be able to just assume it's there.

Reply:
right on.